docs/api/query-chainer.md

```diff
@@ -0,0 +1,79 @@
+<a name="querychainer"></a>
```

Comment:
query chainer is being removed completely in 3.0 so no point in providing docs for it.